.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,3 +12,12 @@
 * [ ] [K8s Upgrade notes updated](https://github.com/sourcegraph/sourcegraph/blob/main/doc/admin/updates/kubernetes.md)
 * [ ] Sister [deploy-sourcegraph-docker](https://github.com/sourcegraph/deploy-sourcegraph-docker) change:
 * [ ] All images have a valid tag and SHA256 sum
+### Test plan
```

Comment:
```suggestion

### Test plan
```

No idea if it matters to the rendered output, just looks nice :)

Reply:
ah I didn't notice this - I'll see if I can update the batch change as well 🤔